docs/release-notes/2.35.0.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Percona Monitoring and Management <version>
+about: Release Notes Template for PMM
+
+---
+
+| **Release date:** | [Month DD, YYYY]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+
+## Release Highlights
+
```

Comment:
- PMM-11302 (https://jira.percona.com/browse/PMM-11302) Ability to add backup schedule while creating DB clusters in DBaaS
- PMM-11301 (https://jira.percona.com/browse/PMM-11301) Ability to restore cluster from backup stored on S3 while creating DB clusters in DBaaS